Sum of digits of a number.py

```diff
@@ -1,7 +1,13 @@
-q=0
-n=int(input("Enter Number: "))
-while(n>0):
- r=n%10
- q=q+r
- n=n//10
+q=0   # Initially we assigned 0 to "q", to use this variable for the summation purpose below.
```

Comment:
You can improve it further.
- By taking number as a string.
- Checking for its numerical value.
- If good, then using split or loop function.
- And at the same time adding them.

This program lacks, 
Error management.
In real world suppose I gave it string input, then it would just crash, right?
What if I gave it a float number? Because float numbers has ' . ' (dot) in them, then again it would crash.
So, being pragmatic is the approach.

Reply:
Yes! you said right. I'll do that today.